### ISSUE TITLE
Colon inside a Literal value produces parsing error

### ISSUE THREAD

**Issue by shamrin** [microsoft/pyright#257]
**Describe the bug**
Colon inside `Literal` value is a parsing error.

**To Reproduce**
```python
from typing_extensions import Literal
literal: Literal["en:us"]
```

**Expected behavior**
I expect not to get a parsing error.

**Screenshots or Code**
<img width="403" alt="image" src="https://user-images.githubusercontent.com/510678/65429888-9cf27a80-de1f-11e9-9dc6-60952cff9804.png">

**VS Code extension or command-line**
Name: Pyright
Id: ms-pyright.pyright
Description: VS Code static type checking for Python
Version: 1.0.64
Publisher: ms-pyright
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ms-pyright.pyright

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is fixed in 1.0.65, which I just published.
